dimod/reference/composites/structure.py

```diff
@@ -104,7 +104,7 @@ def sample(self, bqm, **sample_kwargs):
             >>> edges = [(0, 1)]
             >>> composed_sampler = dimod.StructureComposite(dimod.ExactSolver(),
             ...                          nodes, edges)
-            response = composed_sampler.sample_ising({0: 1, 1: 1}, {})
+            >>> response = composed_sampler.sample_ising({0: 1, 1: 1}, {})
```

Comment:
this aught to be "sampleset"

Reply:
For dimod, because it is not sampling, I have been mostly changing to "solutions" or, as here when I was rushing, leaving unchanged for now.

Reply:
Calling it "solutions" is really only valid in the case of the `ExactSolver` and I think it's better to be consistent with `sampleset`. But agree that leaving it as `response` is fine.

Reply:
My typo: I meant "For dimod.ExactSolver, because"